Sources/TuistGenerator/Mappers/AutogeneratedProjectSchemeWorkspaceMapper.swift

```diff
@@ -6,11 +6,13 @@ import TuistGraph
 /// From a given workspace
 public final class AutogeneratedProjectSchemeWorkspaceMapper: WorkspaceMapping { // swiftlint:disable:this type_name
     private let enableCodeCoverage: Bool
+    private let codeCoverageMode: CodeCoverageMode
```

Comment:
Can't this just be:
```suggestion
    private let codeCoverageMode: CodeCoverageMode?
```

Otherwise we might have inconsistent scenarios like `enableCodeCoverage == false` and `codeCoverageMode == .all`?